include/kernel/faulthandler.h

```diff
@@ -7,13 +7,33 @@
 #pragma once
 
 #include <object.h>
+#ifdef CONFIG_KERNEL_MCS
+#include <kernel/sporadic.h>
+#endif
 
 #ifdef CONFIG_KERNEL_MCS
 static inline bool_t validTimeoutHandler(tcb_t *tptr)
 {
     return cap_get_capType(TCB_PTR_CTE_PTR(tptr, tcbTimeoutHandler)->cap) == cap_endpoint_cap;
 }
 
+static inline bool_t validInvalidSchedContextHandler(tcb_t *tptr)
```

Comment:
I don't understand the name of this function

Reply:
It's like `validTimeoutHandler` for the new fault type. In any case, it goes away in the next commit.